lib/checks/tables/same-caption-summary.json

```diff
@@ -5,7 +5,8 @@
     "impact": "minor",
     "messages": {
       "pass": "Content of summary attribute and <caption> are not duplicated",
-      "fail": "Content of summary attribute and <caption> element are identical"
+      "fail": "Content of summary attribute and <caption> element are identical",
+      "incomplete": "Unable to determine if <table> element has children."
```

Comment:
```suggestion
      "incomplete": "Unable to determine if <table> element has a caption."
```

Reply:
Made the change, and took out the period which was not present in other messages.